CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ community.zabbix Release Notes
 
 .. contents:: Topics
 
+v1.9.2
```

Comment:
Please remove any changes to CHANGELOG.rst as it will be generated automatically at the time we decide to release the next version of this collection.
Instead please put all the changes in a file as a changelog fragment in this folder https://github.com/ansible-collections/community.zabbix/tree/main/changelogs/fragments

Reply:
Thanks for removing changes in CHANGELOG.rst. Would you please create a changlog fragment? for reference https://docs.ansible.com/ansible/latest/community/collection_development_process.html#creating-changelog-fragments

Reply:
Sorry about that. I should have paid more attention to your previous request (not entirely awake, I guess).

Also, it's the first time I see this kind of fragments changelog, but I should have read the contribution guidelines before submitting the PR.

Reply:
no problem at all -)